packages/react-scripts/layout/views/layout.ejs

```diff
@@ -39,6 +39,7 @@
       SERVER_DATA.sgBaseUrl = "<%= process.env.SG_BASE_URL %>"
       SERVER_DATA.splitioAuthKey = "<%= process.env.SPLITIO_AUTH_KEY %>"
       SERVER_DATA.geoData = <%- typeof geoData !== 'undefined' ? JSON.stringify(geoData) : JSON.stringify({}) %>;
+      SERVER_DATA.loggedInSession = <%= loggedInSession %>
```

Comment:
if you do the `typeof foo !== 'undefined'` like with geoData and appPathOverride, then it won't throw an error, it'll just be undefined.

That may be a safer solution, just in case an app updates react-scripts without updating snow.

Reply:
`isLoggedIn` is already set so we shouldn't need to worry about that anymore.